### PR TITLE
Milestone 7. Portfolio Project Mobile Menu

### PR DESCRIPTION
**Pull request by otmanTR**
On this PR following changes made;
-Updated linters
-Updated HTML file
-Updated CSS file
-Added JS file
-Added Mobile Menu

### CODE REVIEW

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @otmanTR , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
| |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Great commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
+
+document.getElementById('l3').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
```
Comment (roshan-bajgain) [inline index.js:29]: - [ ] As requested by the previous reviewer, your navbar should take the height of 100vh so whenever the user clicks the hamburger buttons it should take the whole height and the position of the navbar should be fixed. Please fix the opacity as in the Figma design. See the image below for reference
### your design

your

### Figma design
fimga

File: index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+document.getElementById('l2').onclick = () => {
+  document.getElementById('navulitems').classList.remove('hamburger');
+  document.getElementById('hamburger').classList.remove('hamx');
+  document.getElementById('x').classList.add('hamx');
+};
```
Comment (roshan-bajgain) [inline index.js:23]: - [ ] It was working properly so the first reviewer did not give the required changes but now it is not working please fix it. You have a horizontal scrolling problem please fix it on mobile screen.
### Error
Screenshot 2023-01-04 175304

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @otmanTR, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- You have fixed the opacity issue
- You have fixed the height issue
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] I cannot see the ```x``` button unless I scroll horizontally as shown below and this is because there is still an overflow problem like the previous reviewer stated. Kindly fix it
Screenshot 2023-01-04 at 14 36 24


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (sja-thedude) [inline index.js:23]: - [ ] Kindly make sure to fix the close button (X) it should be displayed correctly on the mobile menu as suggested by the previous reviewer. Correctly it has a horizontal scroll issue. Kindly see the below video and where the button (X) should be shown in the last.

https://user-images.githubusercontent.com/70269507/210591415-0ab27b82-5a4a-471b-9f46-f41ebeee8d76.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @otmanTR, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far
✔️ Professional Readme

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (otmanTR) [reply index.js:23]: Hello @sja-thedude . I check your's and others' requests and I found it seems different in google chrome and Mozilla Firefox
.  Here are the screenshots for each margin and browser.  
Screenshot (3)
Screenshot (4)
Screenshot (5)
Screenshot (6)
Screenshot (7)
Screenshot (8)
As you could see, I think you all check google chrome the live version but I prepared it in Mozilla, so its seen different for us. I already used my extra code review for this project. I could make it for google chrome and submit but on that time I won't have any code review left. Could you approve my project or should I write this to support?

Comment (roshan-bajgain) [reply index.js:23]: Hello @otmanTR, Please fill out this form [Link](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by bimbolabuari — COMMENTED** [review]
Hello @otmanTR  👋,

_This is an additional review due to the second opinion request._

- Nice implementation of the mobile menu
   - However, I tried  the web app on chrome and firefox, and although the close icon is visible, it is not responsive as required and there is a horizontal issue with the application.


https://user-images.githubusercontent.com/66112760/210627612-5362574d-c052-4de9-8cf0-51eac7cfb13b.mov

- Kindly implement the previously requested changes in order to get approved.

Keep rocking 🚀!

**Comment by otmanTR** [comment]
Hello @bimbolabuari,
thank you for your answer. But I am in dark about the responsivity of the close icon. How should the close icon give responsive, doesn't it close the window? Also, I don't know how to fix the horizontal issue. If you could check my CSS code, you would see my width property is defined as width %. Please could you show me the wrong line of my code and I would change it? Thanks.

**Comment by bellom** [comment]
@otmanTR

### This is a manual review

Great job fixing the issue.

Note: It will be approved on your dashboard as soon as possible.

# Approved

Optional: Always try to fix the linter's error before asking for a review.

Cheers and Happy coding!

**Pull request merged**